gremlin-dotnet/src/Gremlin.Net/Driver/Connection.cs

```diff
@@ -126,11 +130,14 @@ private void Parse(byte[] received)
             {
                 if (_callbackByRequestId.TryRemove(receivedMsg.RequestId, out var responseHandler))
                 {
-                    responseHandler.HandleFailure(e);
+                    responseHandler?.HandleFailure(e);
                 }
             }
         }
 
+        private static void ThrowMessageDeserializedNull() =>
+            throw new InvalidOperationException("Received data deserialized into null object message. Cannot operated on it.");
```

Comment:
Just a small grammatical nitpick, but it should be _operate on it_ (without a "d") afaik.